crates/topos-tce-storage/src/client.rs

```diff
@@ -94,14 +34,37 @@ impl StorageClient {
     pub async fn get_pending_certificates(
```

Comment:
Would tce grpc API to get pending certificates work now?

Reply:
I need to work on the tests to validate that it is